game.js

```diff
@@ -0,0 +1,80 @@
+let playerPoints = 0;
+let computerPoints = 0;
+let choices = ["Rock", "Paper", "Scissors"];
+
+// Here we taking the choice of computer
+getComputerChoice = () => choices[Math.floor(Math.random()*choices.length)]
+
+// Here we are Playing a round of game between Player and Computer
+playRound = (playerSelection, computerSelection) => {
+    switch(playerSelection.toLowerCase()){
+        case 'rock':
+            switch(computerSelection.toLowerCase()){
```

Comment:
extract computerSelection.toLowerCase() in a separate variable

Reply:
Okay.Done